### DISCUSSION TITLE
This email address already exists in this module.

### DISCUSSION THREAD

**Discussion by Jeroenleender** [YetiForceCompany/YetiForceCRM#15685] [How to] [answered]
When having two contacts sharing the same email address it's impossible to update either contact because of this warning.

I understand that normally a contact shouldn't have the same email as another contact but i have several situations where that happens.

Those contacts where created using import which works fine without any errors.
Is there a way to bypass the check for duplicate email?

Thanks in advance!

**Comment by rskrzypczak** [comment, marked as answer]
Here:
![image](https://user-images.githubusercontent.com/10221999/151959084-1f8bef3e-8bfc-4cae-8081-c22faf52038a.png)
but from GUI you will disable it both for the Accounts and Contacts module.

**Reply by Jeroenleender** [reply]
Works as a charm, thx :)
